livingatlas/solr/conf/managed-schema

```diff
@@ -361,6 +361,16 @@
   <field name="waterBody"  type="string" docValues="true" indexed="true"/>
   <field name="year"  type="int" docValues="true" indexed="true"/>
 
+  <!-- Clustering (Duplicate detection) -->
+  <field name="isClustered" type="boolean" docValues="true" indexed="true"/>
+  <field name="clusteredWith" type="string" docValues="true" indexed="true" multiValued="true"/>
```

Comment:
Be aware that I found examples where we have 18000+ clustering records due to the nature of data. In that example it was gut analysis from intestines, where the metadata was all about the host organism collections, so all the gut samples clustered with everything at the same location.

Reply:
Thanks for this. Ive added a configurable `candidatesCutoff` which is used by the pipeline to drop a cluster of occurrences over a certain size. This is defaults to 50.